### PR TITLE
Update the "Deploy Kubernetes monitoring with Netdata" doc

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

This PR is related with netdata/documentation#22, it includes:

- [X] Applied the Task template, (Prerequisites, Numbered Steps, Expected Results)
- [X] Grouped the two cases, "New Installations" and "Existing Installations" in a Tab element, and renamed the section to "Deploy Netdata on your Kubernetes Cluster". I judged that the "Install the Netdata Helm chart" section could be renamed to just "Add the Netdata Helm repository" as a step for new installations, and then proceed to install as normal
- [X] Added the Cloud options to the scripts, along with the proper messages in the two cases
- [X] Fixed all the links in the file to be absolute GitHub links

### CODE REVIEW

File: packaging/installer/methods/kubernetes.md [hunk 1]
```diff
@@ -19,94 +20,119 @@ The Helm chart installs one `parent` pod for storing metrics and managing alarm
...
--   The [Helm package manager](https://helm.sh/) v3.0.0 or newer on the same administrative system.
+- The [Helm package manager](https://helm.sh/) v3.0.0 or newer on the same administrative system.
+- A Netdata Cloud account with a Space to claim the cluster to.
 
-## Install the Netdata Helm chart
+## Deploy Netdata on your Kubernetes Cluster
 
-We recommend you install the Helm chart using our Helm repository. In the `helm install` command, replace `netdata` with
-the release name of your choice.
+To start [Kubernetes monitoring](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/kubernetes.md), you must first install Netdata, and then
```
Comment (cakrit) [inline packaging/installer/methods/kubernetes.md:36]: Too subtle to have the link here. Make it more explicit with a sentence in the introductory line. (something like to see what you'll get, read xxx).

Note also that the linked file has an invalid link back to this one AND repeats installation prerequisites instructions. Please add another PR to fix that.

File: packaging/installer/methods/kubernetes.md [hunk 2]
```diff
@@ -19,94 +20,119 @@ The Helm chart installs one `parent` pod for storing metrics and managing alarm
...
+[connect](https://github.com/netdata/netdata/blob/master/claim/README.md) your Kubernetes cluster to [Netdata Cloud](https://app.netdata.cloud). The connection process securely
```
Comment (cakrit) [inline packaging/installer/methods/kubernetes.md:37]: No. This link isn't specific to K8s. the section specific to K8s links back here.  https://github.com/netdata/netdata/blob/master/packaging/installer/methods/kubernetes.md#connect-your-kubernetes-cluster-to-netdata-cloud

So we've created a circle of providing no information!

File: packaging/installer/methods/kubernetes.md [hunk 3]
```diff
@@ -19,94 +20,119 @@ The Helm chart installs one `parent` pod for storing metrics and managing alarm
...
+  --set child.claiming.enabled=true 
+  --set child.claiming.token=YOUR_CLAIM_TOKEN 
+  --set child.claiming.rooms=YOUR_ROOM_ID_A,YOUR_ROOM_ID_B
+  ```
 
-First, find the script to run an `helm install` command. You can get it by clicking on your Space's dropdown, then **Manage your Space**.
-Click the **Nodes** tab and select the environment your node is running, in this case **kubernetes**, to reveal the script for your Space in Netdata Cloud. You need the `TOKEN`
-and `ROOM` values.
+  :::note
+  If you plan to also Claim the node to Netdata Cloud,
```
Comment (cakrit) [inline packaging/installer/methods/kubernetes.md:65]: Well, it's a bit strange to give them a command that has the tokens and then say that IF they want to connect to cloud they should xyz. 

This isn't the same as the one line installer though, where we give them additional options in this doc. The full reference is actually on github.com/netdata/helmchart/README.md 
So:
- Give them the simple the helm install command. 
- For claiming, tell them to get their info directly from "Add nodes" in app.netdata.cloud. 
- For more installation options direct them to helmchart/README  (we need to add helmchart to learn as well, obviously).

**Review by cakrit — CHANGES_REQUESTED** [review]

File: packaging/installer/methods/kubernetes.md (hunk 2 above)
Comment (Ancairon) [reply packaging/installer/methods/kubernetes.md:37]: I will remove the link and make it say "To start monitoring Kubernetes", that way it reads in a more natural way

Comment (Ancairon) [reply packaging/installer/methods/kubernetes.md:37]: Will remove the link to the Claim README.

Comment (Ancairon) [reply packaging/installer/methods/kubernetes.md:37]: I saw that a while back https://github.com/netdata/netdata/pull/11378 was merged, changing "claim" with "connect", what's the right term to use?

**Comment by Ancairon** [comment]
@cakrit [this PR](https://github.com/netdata/netdata/pull/11378) changed the term "claim" to "connect", which one should we use?

**Comment by cakrit** [comment]
We use "connect" not claim

**Review by cakrit — APPROVED** [review]

File: packaging/installer/methods/kubernetes.md [hunk 4]
```diff
@@ -8,105 +8,130 @@ learn_topic_type: "Tasks"
...
+import TabItem from '@theme/TabItem';
+
 # Deploy Kubernetes monitoring with Netdata
 
-This document details how to install Netdata on an existing Kubernetes (k8s) cluster. By following these directions, you
-will use Netdata's [Helm chart](https://github.com/netdata/helmchart) to create a Kubernetes monitoring deployment on
-your cluster.
+This document details how to install Netdata on an existing Kubernetes (k8s) cluster, and connect it to Netdata Cloud.  
+
+To see what our Netdata Cloud Kubernetes visualizations have to offer, read the[Kubernetes visualizations Concept](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/kubernetes.md)
```
Comment (cakrit) [inline packaging/installer/methods/kubernetes.md:18]:
```suggestion
To see what our Netdata Cloud Kubernetes visualizations have to offer, read the [Kubernetes visualizations Concept](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/kubernetes.md)
```

File: packaging/installer/methods/kubernetes.md [hunk 5]
```diff
@@ -8,105 +8,130 @@ learn_topic_type: "Tasks"
...
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Deploy Kubernetes monitoring with Netdata
 
-This document details how to install Netdata on an existing Kubernetes (k8s) cluster. By following these directions, you
-will use Netdata's [Helm chart](https://github.com/netdata/helmchart) to create a Kubernetes monitoring deployment on
-your cluster.
+This document details how to install Netdata on an existing Kubernetes (k8s) cluster, and connect it to Netdata Cloud.
```
Comment (cakrit) [inline packaging/installer/methods/kubernetes.md:16]: add to the end of this sentence add "See what you'll get" or something similar and remove the next sentence.

File: packaging/installer/methods/kubernetes.md [hunk 6]
```diff
@@ -8,105 +8,130 @@ learn_topic_type: "Tasks"
...
 # Deploy Kubernetes monitoring with Netdata
 
-This document details how to install Netdata on an existing Kubernetes (k8s) cluster. By following these directions, you
-will use Netdata's [Helm chart](https://github.com/netdata/helmchart) to create a Kubernetes monitoring deployment on
-your cluster.
+This document details how to install Netdata on an existing Kubernetes (k8s) cluster, and connect it to Netdata Cloud.  
+
+To see what our Netdata Cloud Kubernetes visualizations have to offer, read the [Kubernetes visualizations Concept](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/kubernetes.md)
+
+By following these directions, you will use Netdata's [Helm chart](https://github.com/netdata/helmchart/blob/master/charts/netdata/README.md) to create a Kubernetes monitoring deployment on your cluster.
```
Comment (cakrit) [inline packaging/installer/methods/kubernetes.md:20]: Not needed

File: packaging/installer/methods/kubernetes.md [hunk 7]
```diff
@@ -8,105 +8,130 @@ learn_topic_type: "Tasks"
...
 
 The Helm chart installs one `parent` pod for storing metrics and managing alarm notifications, plus an additional
```
Comment (cakrit) [inline packaging/installer/methods/kubernetes.md:22]:
```suggestion
The [Netdata Helm chart](https://github.com/netdata/helmchart/blob/master/charts/netdata/README.md) installs one `parent` pod for storing metrics and managing alarm notifications, plus an additional
```

File: packaging/installer/methods/kubernetes.md [hunk 8]
```diff
@@ -8,105 +8,130 @@ learn_topic_type: "Tasks"
...
--   The [Helm package manager](https://helm.sh/) v3.0.0 or newer on the same administrative system.
+- The [Helm package manager](https://helm.sh/) v3.0.0 or newer on the same administrative system.
+- A Netdata Cloud account with a Space to connect the cluster to.
 
-## Install the Netdata Helm chart
+## Deploy Netdata on your Kubernetes Cluster
 
-We recommend you install the Helm chart using our Helm repository. In the `helm install` command, replace `netdata` with
-the release name of your choice.
+First, you need to install Netdata, and then connect your Kubernetes cluster to [Netdata Cloud](https://app.netdata.cloud).
```
Comment (cakrit) [inline packaging/installer/methods/kubernetes.md:40]: You seem to have forced a two step process out of something that can be done in one step. We'll talk.

**Review by cakrit — CHANGES_REQUESTED** [review]

**Review by cakrit — APPROVED** [review]

**Pull request merged**